hyrax.gemspec

```diff
@@ -26,6 +26,12 @@ EOF
   spec.version       = Hyrax::VERSION
   spec.license       = 'Apache2'
 
+  # Be pessimistic, rails does not follow sem-ver conventions and
+  # minor version releases can include breaking changes; see
+  # http://guides.rubyonrails.org/maintenance_policy.html
+  # https://github.com/projecthydra-labs/hyrax/issues/1028
+  spec.add_dependency 'rails', '>= 5.0.0', '< 5.2'
```

Comment:
I guess the point to add is what do you want to maintain?  Do you want to take the time to track rails releases and manage the upgrade path proactively, which avoid spurious reactive responses to failures whenever someone tries to use hyrax with an un-tested version of rails.  Or, do you want to be reacting to any number of "bug" reports from users of hyrax trying to use it with a rails version that was never tested?  The latter 'reactive' maintenance seems like more work to me and possibly work that never gets done in a planned work cycle and drags developers away from planned work to try to put out fires.